web/client-api/src/main/java/io/deephaven/web/client/api/CoreClient.java

```diff
@@ -77,12 +79,19 @@ public String getServerUrl() {
     }
 
     public Promise<String[][]> getAuthConfigValues() {
-        return getConfigs(
-                // Explicitly creating a new client, and not passing auth details, so this works pre-connection
-                c -> new ConfigServiceClient(getServerUrl(), CLIENT_OPTIONS).getAuthenticationConstants(
-                        new AuthenticationConstantsRequest(),
-                        c::apply),
-                AuthenticationConstantsResponse::getConfigValuesMap);
+        return ideConnection.getConnectOptions().then(options -> {
+            BrowserHeaders metadata = new BrowserHeaders();
```

Comment:
I'm a little confused... you're setting the headers in this metadata, but then this metadata isn't getting used anywhere?

Reply:
Fixed.